lib/material/internal/material_textinput_state.flow

```diff
@@ -94,8 +95,10 @@ export {
 		popup : Maybe<MInputPopupState>,
 		autocomplete : Maybe<MInputAutocompleteState>,
 		error : Maybe<MInputErrorState>,
+		errorId : string,
```

Comment:
Do not extra-populate MInputState, we already have MInputErrorState and MInputHelperState sub-structures.